src/main/webapp/app/view/project/project.routes.ts

```diff
@@ -3,31 +3,28 @@ import { Routes } from '@angular/router';
 import { ProjectResolve } from 'app/entities/project/project.route';
 import { UserRouteRoleAccessService } from 'app/core/auth/user-route-role-access-service';
 
-import { ProjectComponent } from 'app/view/project/project.component';
 import { ScreenplayComponent } from 'app/view/project/screenplay/screenplay.component';
 
-import { ROLE_ADMIN, ROLE_SECRETARY } from 'app/shared/constants/role.constants';
+import { ROLE_ADMIN, ROLE_CONTRIBUTOR, ROLE_SECRETARY, ROLE_VIEWER } from 'app/shared/constants/role.constants';
 
 export const PROJECT_ROUTES: Routes = [
   {
     path: ':projectId',
-    component: ProjectComponent,
-    data: {
-      defaultSort: 'id,asc',
-      pageTitle: 'eventPlannerApp.project.overview.tabTitle',
-      roles: [ROLE_ADMIN, ROLE_SECRETARY],
-    },
-    canActivate: [UserRouteRoleAccessService],
-    resolve: {
-      project: ProjectResolve,
-    },
+    pathMatch: 'full',
+    redirectTo: ':id/admin',
+  },
+  {
+    path: ':projectId/admin',
+    loadChildren: () => import('./admin/project-admin.module').then(m => m.EventPlannerProjectAdminModule),
```

Comment:
Auf der `/admin` Seite sollte nur `ROLE_ADMIN, ROLE_SECRETARY` zugriff haben

Reply:
das ist in den `project-admin.routes.ts` geregelt. da man hier nicht im kontext admin ist, kann man eigentlich auch nicht unbedingt wissen ob man zugriff hat oder nicht. ich denke, das passt dort.